### ISSUE TITLE
Conflict with Github's Dark color scheme

### ISSUE THREAD

**Issue by ghost** [Eclipse-Community/r3dfox-old#224]
The Github online text editor, to be clear. When you select any text, you will only see the red background but not the text. This will not happen with the rest of Github when the text is still visible in white color.

**Comment by the-r3dacted** [comment]
Yeah, I'm not sure if I like how the color overrides every websites text select color, but I'm not sure how to do it any other way. I might experiment with it for next release, or perhaps even remove it.

**Comment by ghost** [comment]
I switched to Firefox-for-windows-7. Goodbye.

https://github.com/e3kskoy7wqk/Firefox-for-windows-7

**Comment by the-r3dacted** [comment]
Should be fixed with https://github.com/Eclipse-Community/r3dfox/commit/3a717e51f9a482eebe38676fe9aa8fb961fcd2e3
